pkg/network/status/status_test.go

```diff
@@ -197,8 +198,28 @@ func TestProbeLifecycle(t *testing.T) {
 	}
 }
 
-func TestCancellation(t *testing.T) {
-	ia := iaTemplate.DeepCopy()
+func TestProbeListerFail(t *testing.T) {
+	ing := ingTemplate.DeepCopy()
+	ready := make(chan *v1alpha1.Ingress)
+	prober := NewProber(
+		zaptest.NewLogger(t).Sugar(),
+		fakeProbeTargetLister{}, // a ProbeTargetLister that always errors
```

Comment:
perhaps the name should be `failingLister`?

Reply:
Done